js/src/features/human-resouce/components/Absence/Abcence.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from "react";
+import { List, Tag, Tooltip } from "antd";
+import {
+  CheckCircleOutlined,
+  CloseCircleOutlined,
+  EditOutlined,
+} from "@ant-design/icons";
+import { Absence as AbsenceModel } from "HumanResourceModels";
+
+interface Props {
+  absence: AbsenceModel;
+  canEdit: boolean;
+  approve: (id: string) => any;
+  remove: (id: string) => any;
+  openUpdateModal: (id: string) => any;
+}
+
+export const Absence: React.FC<Props> = ({
+  absence,
+  canEdit,
+  approve,
+  remove,
+  openUpdateModal,
+}) => {
+  const handleApprove = useCallback(() => {
+    if (!absence.isApproved) {
+      approve(absence.id);
+    }
+  }, [absence, approve]);
+
+  const handleRemove = useCallback(() => {
+    remove(absence.id);
+  }, [absence, remove]);
+
+  const handleOpenUpdate = useCallback(() => {
+    openUpdateModal(absence.id);
+  }, [absence, openUpdateModal]);
+
+  const title = `${absence.startedAt} ${absence.endedAt}`;
+  const description = `${absence.reason}: ${absence.comment}`;
+  const content = absence.isApproved ? (
+    <Tag color="green">approved</Tag>
+  ) : (
+    <Tag color="red">Not approved</Tag>
+  );
+
+  const actions = [
+    <Tooltip title="Remove">
```

Comment:
May be better for UX rename this action to Dismiss? It will looks more humanized.

Reply:
I think `remove` is fine because it actually removes items from DB

Reply:
What we do in DB it doesn't matter. If we'll change behavior and move this absence to history DB table it shouldn't have the effect for UI